day-four-challenge/secure-container/getPasswords.js

```diff
@@ -0,0 +1,33 @@
+const { map, sort, not, uniq, equals, unfold } = require('ramda');
+
+const MIN = 359282;
+const MAX = 820401;
+const PASSWORD_MIN_LENGTH = 6;
+
+const diff = (a, b) => { return a - b };
+
+const digitsIncreaseOrStayTheSameFromLeftToRight = array => (
+  equals(sort(diff, array), array)
+);
+
+const hasAtLeastTwoRepeatingDigits = (array) => (not(equals(uniq(array), array)))
+
+const isASixDigitNumber = (array) => array.length === PASSWORD_MIN_LENGTH;
+
+const meetsPasswordRequirements = (num) => {
+  const numStringArray = num.toString().split('');
+  const numArray = map(parseInt, numStringArray);
+
+  return isASixDigitNumber(numArray) && digitsIncreaseOrStayTheSameFromLeftToRight(numArray) && hasAtLeastTwoRepeatingDigits(numArray)
+}
+
+function findPasswords(num = MIN, max = MAX) {
```

Comment:
Are there cases where we'd want to be able to use this function without specifying numbers? If not, we might be able to nix the `MIN` and `MAX` static constants.

Reply:
I kept these as static constants so that I don't have to declare them in my index.js file which is where all my other Advent of Code solutions will go in the future. I've removed these constants from my index.js file here 339efc241058690fab07d5ad876cfbf103ec9729.

Reply:
Nice!